src/assets/js/pages/NewEventPage/index.js

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {showLoading, hideLoading} from 'react-redux-loading-bar';
+import createValidator from './validate';
+import {registerNewEvent} from '~/data/Api'
+import NewEventForm from './components/NewEventForm';
+import {UncontrolledAlert} from 'reactstrap';
+
+class NewEventPage extends React.Component {
+  static propTypes = {
+    showLoading: PropTypes.func.isRequired,
+    hideLoading: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      error: null,
+      event: null
+    }
+
+    this.createNewEvent = this.createNewEvent.bind(this);
+  }
+
+  createNewEvent(event) {
+    registerNewEvent(event)
+      .then((res) => {
+        console.log("res");
+        this.setState({
+          error: null,
+          event: res
+        });
+      })
+      .catch((err) => {
+        this.setState({
+          error: err.message,
+          event: null
+        })
+      });
+  }
+
+  renderAlert() {
+    if (this.state.event) {
+      return (
+        <UncontrolledAlert color="success">
+          Successfully Created The <b>{this.state.event.name}</b> Event
```

Comment:
Maybe "Successfully Created HackXX 2018" rather than "Successfully Create The HackXX 2018 Event" ?
Debatable, though.
If including "The", probably lowercase "the" at least.